### DISCUSSION TITLE
gmc/blog/tutorials/guidelines-for-community-posts/

### DISCUSSION THREAD

**Discussion by giscus** [Gothic-Modding-Community/gmc#132] [_]
# gmc/blog/tutorials/guidelines-for-community-posts/

Gothic Modding Community is a collection of helpful resources for modding Gothic and Risen games.

https://kamilkrzyskow.github.io/gmc/blog/tutorials/guidelines-for-community-posts/

**Comment by kamilkrzyskow** [comment]
Bruh, provide a TLDR
